assets/css/main.scss

```
@@ -37,4 +37,8 @@
 
 .deprecated-link {
   color: gray;
+}
+
+.table-of-contents ul {
+  padding-inline-start: 15px;
```

Not sure if there is a strong reason to use `padding-inline-start` over `padding-left` , which is also [supported in Internet Explorer](https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start#browser_compatibility)?

```suggestion
  padding-left: 15px;
```

---

I picked that because it's what showed up in the Chrome inspector, `padding-left` is the one I'm more familiar with, will go with that